### ISSUE TITLE
Remove solid shaded box on clickable items throughout app. 

### ISSUE THREAD

**Issue by MenaMachado** [CDH-Studio/I-Talent#437]
**Describe the bug**

In dev, there is a shaded box that appears when users click or tab on actionable items. The colour of the shaded box varies. Sometimes it is dark grey, light green, dark green. 

This should be fixed for UAT. 

**Expected behavior**
As per observations on ISEDWeb.

- For accessibility, when tabbing, use a rectangle (non rounded edges)  solid light grey or dotted box. Ensure that it completely surrounds the text. (see example below from ISEDWeb

- When mouse clicking, remove box completely. 

**Screenshots**
Example of outline 
![image](https://user-images.githubusercontent.com/49656364/85900826-773f3b00-b7ce-11ea-9ebc-3cb12c650215.png)

See shaded box on basic search
![image](https://user-images.githubusercontent.com/49656364/85900531-fed87a00-b7cd-11ea-9af7-8a7f2750dae5.png)
